lighthouse-cli/types/types.ts

```diff
@@ -1,13 +1,25 @@
 interface AuditResult {
+  rawValue: boolean|number;
+  displayValue?: string;
+  debugString?: string;
+  score?: boolean|number;
+  optimalValue: number|string;
+  extendedInfo?: {
+    value: string;
+    formatter: string;
+  };
+}
+
+interface AuditFullResult {
+  rawValue: boolean|number;
   displayValue: string;
-  debugString: string;
-  score: number;
+  debugString?: string;
+  score: boolean|number;
   error?: boolean;
   description: string;
   name: string;
   category: string;
   helpText?: string;
-  requiredArtifacts?: Array<string>;
```

Comment:
this is for the `AuditResult`, not `Audit`. I'm not sure why `requiredArtifacts` was on this in the first place